src/Avalonia.Diagnostics/DevTools.xaml.cs

```diff
@@ -14,12 +14,21 @@
 
 namespace Avalonia
 {
-	public static class WindowExtensions
+	public static class DevToolsExtensions
 	{
-		public static void AttachDevTools(this Control control)
+		public static void AttachDevTools(this TopLevel control)
 		{
-			Avalonia.Diagnostics.DevTools.Attach((TopLevel)control.GetVisualRoot());
+			Avalonia.Diagnostics.DevTools.Attach(control);
 		}
+
+	    public static void AttachDevToolsToTopLevelOnVisualTreeAttachment(this Control control)
```

Comment:
It's also very convenient to use for `Popup` debugging. User doesn't have *any* control over `PopupRoot`, so one needs to subscribe to events, etc. Better have a built-in API for that